src/hooks/useHelpScout.js

```diff
@@ -0,0 +1,35 @@
+import React, { useContext, useState } from 'react'
```

Comment:
Rather than in `hooks/` perhaps we can leave this in `components/HelpScoutBeacon/` since we're usually leaving most hooks with their logical components.

Reply:
Sure, makes sense.